VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,36 @@
+# Video 5: Open Governance
+Hello everyone! Thank you for joining us in today's video. Today, we will talk about Open Governance and why AsyncAPI is committed to this model. 
+
+Let's get started!
+
+## What is Open Governance
+Open Governance is a model of management based on transparency, collaboration, and ethics which results in clearer accountability and management. Driven by the participation of all the people that make up a project, company, or institution, it promotes a democratic system with inclusive growth.
```

Comment:
majority of content from this section talks about open-source in general. Not much about open governance itself.

have a look at:
- https://www.ibm.com/blogs/cloud-computing/2016/10/27/open-source-open-governance/
- https://www.whitesourcesoftware.com/resources/blog/best-practices-for-open-source-governance/
- https://www.redhat.com/en/blog/understanding-open-source-governance-models

I mean we should not explain what open source is and why it is done, but open governance specifically